dark/graphics.py

```diff
@@ -703,3 +721,41 @@ def scatterAlign(seq1, seq2, window=7):
     plt.title('Dot plot using window size %i\n(allowing no mis-matches)' %
               window)
     plt.show()
+
+
+def plotAAProperties(sequence, propertyNames, showPlot=True):
+    """
+    Plot amino acid property values for a sequence.
+
+    @param sequence: An C{AARead} (or a subclass) instance.
+    @param propertyNames: An iterable of C{str} property names (each of which
+        must be a key of a key in the dark.aa.PROPERTY_DETAILS dict).
+    @param showPlot: If C{True}, display the plot. Passing C{False} is useful
+        in testing.
+    @raise ValueError: If an unknown property is given in C{propertyNames}.
+    @return: The return value from calling dark.aa.propertiesForSequence:
+        a C{dict} keyed by (lowercase) property name, with values that are
+        C{list}s of the corresponding property value according to sequence
+        position.
+    """
+    MISSING_AA_VALUE = -1.1
```

Comment:
Is there a reason why this is different from the default in propertiesForSequence above?
And why the uppercase?